### DISCUSSION TITLE
PHABMACS

### DISCUSSION THREAD

**Discussion by varshasathya** [eclipse/mosaic#179] [Q&A] [answered]
Hi, I was trying to explore more on the PHABMACS simulator, but I couldn't find much information regarding the same other than the fact that it is available with the Extended Simulator. Is there any dedicated site or whitepapers available on the PHABMACS simulator?

Any detailed piece of information about PHABMACS would be helpful.

**Comment by kay-ma** [comment, marked as answer]
Hi, we are currently moving from PHABMACS 2.0 to PHABMACS 3.0. and we haven't updated the docs yet. As the docs we have are deprecated, we don't want to put them online. I will send you a personal mail with some material. 

Here you can find some papers about the models used in PHABMACS:
Massow, Kay, and Ilja Radusch. "**A rapid prototyping environment for cooperative advanced driver assistance systems.**" Journal of Advanced Transportation 2018 (2018).

about how scenarios are defined and the concept of features / skills
Massow, Kay, et al. "**Scenario Definition for Prototyping Cooperative Advanced Driver Assistance Systems.**" 2020 IEEE 23rd International Conference on Intelligent Transportation Systems (ITSC). IEEE, 2020.

**Reply by varshasathya** [reply]
Received the mail Kay. Thank you so much for the insights.
